packages/docs-site/src/components/clickToCopy.tsx

```diff
@@ -60,7 +57,7 @@ export class ClickToCopy extends React.Component<IClickToCopyProps, IClickToCopy
         return (
             <div
                 {...removeNonHTMLProps(this.props, ["copiedClassName", "value"], true)}
-                className={classNames("docs-clipboard", className, { [copiedClassName]: this.state.hasCopied })}
+                className={classNames("docs-clipboard", className, { [copiedClassName!]: this.state.hasCopied })}
```

Comment:
`strictNullChecks` assert that something is defined (remove `| undefined` basically). necessary because most props are marked optional so they're not required in the JSX.